src/Sulu/Bundle/CategoryBundle/Resources/public/js/main.js

```diff
@@ -22,35 +22,89 @@ define({
 
         'use strict';
 
-        var sandbox = app.sandbox;
+        var sandbox = app.sandbox,
+            CATEGORIES_LANGUAGE = 'categoryLocale',
```

Comment:
Would also call this one `CATEGORIES_LOCALE`, and if you are using it multiple times it would probably be worth putting this into the `Config` object, don't you think?